README.md

```diff
@@ -102,74 +110,69 @@ buildozer android debug deploy
 buildozer setdefault android debug deploy run
 ```
 
-
 ## Usage
 
-```
-Usage:
-    buildozer [--profile <name>] [--verbose] [target] <command>...
-    buildozer --version
+```yml
+Usage: buildozer [--profile <name>] [--verbose] [target] <command>...
+  buildozer --version
 
 Available targets:
-    android        Android target, based on python-for-android project
-    ios            iOS target, based on kivy-ios project
+  android        Android target, based on python-for-android project
+  ios            iOS target, based on kivy-ios project
 
 Global commands (without target):
-    distclean          Clean the whole Buildozer environment
-    help               Show the Buildozer help
-    init               Create an initial buildozer.spec in the current directory
-    serve              Serve the bin directory via SimpleHTTPServer
-    setdefault         Set the default command to run when no arguments are given
-    version            Show the Buildozer version
-
-Target commands:
-    clean      Clean the target environment
-    update     Update the target dependencies
-    debug      Build the application in debug mode
-    release    Build the application in release mode
-    deploy     Deploy the application on the device
-    run        Run the application on the device
-    serve      Serve the bin directory via SimpleHTTPServer
+  distclean          Clean the whole Buildozer environment
+  help               Show the Buildozer help
+  init               Create an initial buildozer.spec in the current directory
+  serve              Serve the bin directory via SimpleHTTPServer
+  setdefault         Set the default command to run when no arguments are given
+  version            Show the Buildozer version
+
+Target commands: clean      Clean the target environment
```

Comment:
I guess `clean         Clean the target environment` was expected to be on a new line. I'm right?

Reply:
Yes, I fixed this